### PR TITLE
IMP gitignore: add missing entry

### PR DESCRIPTION
**Pull request by bealdav**

### CODE REVIEW

File: src/.gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 /secrets.docker-compose.yml
 /clear-*
 .home
+*.~lock*  # libreoffice may generate these files while opening file
+*.dump    # db files shouldn't be taken account by git
```
Comment (PierrickBrun) [inline src/.gitignore:8]:
```suggestion
*.dump    # db files shouldn't be taken into account by git
```

**Review by PierrickBrun — APPROVED** [review]

**Comment by bealdav** [comment]
Maybe other entries ?

File: src/.gitignore [hunk 2]
```diff
@@ -4,3 +4,5 @@
 /secrets.docker-compose.yml
 /clear-*
 .home
+*.~lock*  # libreoffice may generate these files while opening file
+*.dump    # db files shouldn't be taken into account by git
```
Comment (hparfr) [inline src/.gitignore:8]: [here](https://github.com/akretion/docky-odoo-template/blob/main/src/odoo/.gitignore)

Comment (bealdav) [reply src/.gitignore:8]: Why a per user config ? If somebody commits ...lock. file in a project in which I work, add it my user config won't prevent to have these parasite files in commits of the project.
It's a complete standard global for all projects

**Comment by bealdav** [comment]
Merci @hparfr

**Pull request merged**